lib/passport/modules/two_factor_auth.ex

```diff
@@ -4,20 +4,29 @@ defmodule Passport.TwoFactorAuth do
 
   defmacro schema_fields do
     quote do
+      field :tfa_confirmation_token, :string
```

Comment:
?

Reply:
2fa has no confirmation token, it should work just like entering 2fa normally --- user gives us the passphrase and we validate with potp